README.md

```diff
@@ -70,141 +70,6 @@ snow --version
 
 You should now be able to run `snow` and get the CLI message.
 
```

Comment:
I suggest deleting starting with line 11 (Overview), line 19 (Benefits) or line 30 (Limitations).

Or, what do you think of this:

Note text (remove For complete installation...)

Overview

Benefits

For complete installation...

Get involved

Reply:
I removed more but left installation instruction so people can easily get the tool